test/iam.ts

```diff
@@ -110,20 +113,20 @@ describe('IAM', () => {
   });
 
   describe('setPolicy', () => {
-    const policy = {etag: 'ACAB'};
+    const policy = {etag: 'ACAB', bindings: []} as iamTypes.Policy;
 
-    it('should throw an error if a policy is not supplied', () => {
-      assert.throws(() => {
-        iam.setPolicy(util.noop);
-      }, /A policy object is required\./);
-    });
+    // it('should throw an error if a policy is not supplied', () => {
```

Comment:
need suggestion, in setPolicy method policy parameter is required parameter, i think this test case is not required if we giving type to iam variable.
https://github.com/praveenqlogic/nodejs-pubsub/blob/master/src/iam.ts#L234